packages/components/result/src/Result.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="ix-result" :class="className">
+    <div class="ix-result-icon">
+      <slot name="icon">
+        <ix-icon :name="currentIcon"></ix-icon>
+      </slot>
+    </div>
+    <slot name="title">
+      <div v-if="title" class="ix-result-title">{{ title }}</div>
+    </slot>
+    <slot name="subtitle">
+      <div v-if="subtitle" class="ix-result-subtitle">{{ subtitle }}</div>
+    </slot>
```

Comment:
```suggestion
    <div v-if="title || hasTitleSlot" class="ix-result-title">
      <slot name="title">{{ title }}</slot >
    </div>
    <div v-if="subtitle || hasSubtitleSlot" class="ix-result-subtitle">
      <slot name="subtitle">{{ subtitle}}</slot >
    </div>
```
这样会不会更好些？否则用户通过 slot 传进来的样式都不统一

Reply:
我觉得这种可以规定一下，slot传进来的到底是完全交给调用方来管理，还是保留我们自己的一些样式配置

Reply:
保留我们的样式设置，你看其他组件包括，你这个组件的  extra 和 content